apps/backoffice-v2/src/lib/blocks/variants/BlocksVariant/BlocksVariant.tsx

```diff
@@ -0,0 +1,28 @@
+import { ManualReviewBlocks } from '@/lib/blocks/variants/ManualReviewBlocks/ManualReviewBlocks';
+import { KybDemoBlocks } from '@/lib/blocks/variants/KybDemoBlocks/KybDemoBlocks';
+import { DefaultBlocks } from '@/lib/blocks/variants/DefaultBlocks/DefaultBlocks';
+import { FunctionComponent } from 'react';
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+import { WorkflowDefinitionVariant } from '@ballerine/common';
+
+export const BlocksVariant: FunctionComponent<{
+  workflowDefinition: Pick<TWorkflowById['workflowDefinition'], 'variant' | 'config' | 'version'>;
+}> = ({ workflowDefinition }) => {
+  if (
+    workflowDefinition?.version >= 0 &&
```

Comment:
For readability purposes (and assuming we're gonna have more variants in the future), I think it would be best if you extract the conditions in each `if` statement to a function whose name would describe what we're checking for